.github/workflows/check-commit-message.yml

```diff
@@ -0,0 +1,19 @@
+name: check-merge-commit-message-on-PR
+
+run-name: Check merge commit messages by ${{ github.actor }} on ${{ github.ref }}
+on: [pull_request]
+
+jobs: 
+  check_title_and_description:
+    if: github.event_name == 'pull_request' && ! github.event.pull_request.draft
+    name: check PR title and description
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
@Samahu - I tried removing it, but I'm not sure how I'm supposed to get access to the script if it's not there?

Reply:
Yeah I think it make sense.